files/etc/ssl/private/workarea.crt

```diff
@@ -0,0 +1,17 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Seems like you'd want to generate this every time.... like nothing good could come from reusing the same cert every time. Do we have the domain name? You could use lets encrypt if you did.

Reply:
we do, however if a user opts to not choose a domain name and default to the IP, we'll need something there in order for the app to run in production. if someone provides a domain, however, i could probably use `certbot` to get a real SSL cert in there..

Reply:
Hmm ok. Either way, we should take the extra time to not reuse the same cert for all instances.